assignment/src/com/assignmentweek3/fileinputoutput/FileInputOutput.java

```diff
@@ -0,0 +1,32 @@
+package com.assignmentweek3.fileinputoutput;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileInputOutput {
+    /*
+    This is the main method
+     */
+    public static void main(String[] args) throws IOException {
+        File sourcefile = new File("sourcefile.txt");
+        File targetfile = new File("targetfile.txt");
+        FileWriter sourceInput = new FileWriter("sourcefile.txt");
```

Comment:
Don't hard code file names directly. Define constant variables for file names and use those constants inside File constructor.